lib/ansible/modules/cloud/vmware/vmware_portgroup_facts.py

```diff
@@ -98,54 +112,112 @@
 
 
 class PortgroupFactsManager(PyVmomi):
+    """Class to manage Port Group facts"""
     def __init__(self, module):
         super(PortgroupFactsManager, self).__init__(module)
         cluster_name = self.params.get('cluster_name', None)
         esxi_host_name = self.params.get('esxi_hostname', None)
         self.hosts = self.get_all_host_objs(cluster_name=cluster_name, esxi_host_name=esxi_host_name)
+        if not self.hosts:
+            self.module.fail_json(msg="Failed to find host system.")
+        self.policies = self.params.get('policies')
 
     @staticmethod
-    def normalize_pg_info(portgroup_obj):
+    def normalize_pg_info(portgroup_obj, policy_facts):
+        """Create Port Group information"""
         pg_info_dict = dict()
-        pg_info_dict['name'] = portgroup_obj.spec.name
```

Comment:
Removal of this line broke one of my scripts, which was relying on the field "name" to contain the portgroup name.

Reply:
Sorry. I've forgot this attribute. Should have been added to line 133 like the other attribute..